factcast-schema-registry-cli/src/main/resources/site/hugo-theme-docdock-master/exampleSite/content/page-images.md

```diff
@@ -0,0 +1,10 @@
++++
+title = "About images"
+date = "2017-04-24T18:36:24+02:00"
+
+[menu.main]
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [[no-shortcut-reference-link] Use the trailing [] on reference links](https://app.codacy.com/manual/uwe/factcast/pullRequest?prid=4957021)